Makefile

```diff
@@ -34,39 +32,45 @@ docker:
 	docker cp hugo-build:/go/bin/hugo .
 	docker rm hugo-build
 
+govendor:
+	go get -u github.com/kardianos/govendor
+	go install github.com/kardianos/govendor
```

Comment:
FYI

> "Get downloads the packages named by the import paths, along with their
dependencies. It then installs the named packages, like 'go install'." - `go get -h`